Tests/Unit/DependencyInjection/DoctrinePHPCRExtensionTest.php

```diff
@@ -74,7 +73,7 @@ public function testCustomManagerRegistryService()
             'manager_registry_service_id' => 'my_phpcr_registry',
         ));
 
-        /** @var $repositoryFactory DefinitionDecorator */
+        /* @var $repositoryFactory DefinitionDecorator */
```

Comment:
i think it proposed the change because there is ``$repositoryFactory`` used after those lines .. so I will remove that comment .. it didn't mess with other comments that did set a phpdoc for a valid variable name.